raiden/network/transport/matrix/transport.py

```diff
@@ -473,7 +472,7 @@ def whitelist(self, address: Address):
         This may be called before transport is started, to ensure events generated during
         start are handled properly.
         """
-        self.log.debug('Whitelist', address=to_normalized_address(address))
+        self.log.debug('Whitelist', address=pex(address))
```

Comment:
please, revert this, `to_checksum_address` is the serialization format.

Reply:
There is only `to_checksum_address` being used in `/network/transport/matrix` now